api/src/main/java/io/opencensus/tags/TagContextBuilder.java

```diff
@@ -28,13 +28,29 @@
   /**
    * Adds the key/value pair regardless of whether the key is present.
    *
+   * <p>Equivalent to calling {@code put(key, value,
+   * TagMetadata.create(TagTtl.UNLIMITED_PROPAGATION))}.
```

Comment:
Same, to keep backwards-compatibility this (deprecated) method still produces propagating `Tag`s.

Reply:
This should be in the method comment.

Reply:
SG, added to Javadoc.